docs/tutorials/index.rst

```diff
@@ -10,12 +10,14 @@ Models
 ======
 This tutorial presents several random graph models: the Erdos-Renyi (ER) model, degree-corrected ER model,
 stochastic block model (SBM), degree-corrected SBM, and random dot product graph model. These models provide a basis for studying random graphs. All models are shown fit to the same dataset.
+This tutorial also demonstrates how to sample graphs of the same degree sequence using degree preserving edge swaps
```

Comment:
can you update language in this section to reflect that there are now two tutorials instead of one